<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dict_note_add_window</class>
 <widget class="QMainWindow" name="dict_note_add_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>加入复习计划</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dict">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>单词</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sure">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>音标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_more">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>继续添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_explain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>释义</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>分组</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_mask">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_dict">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>461</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit_explain">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
